src/hotspot/share/cds/cdsConfig.hpp

```diff
@@ -71,15 +80,18 @@ class CDSConfig : public AllStatic {
   static const char* dynamic_archive_path()                  { return CDS_ONLY(_dynamic_archive_path) NOT_CDS(nullptr); }
 
   static int num_archives(const char* archive_path)          NOT_CDS_RETURN_(0);
+  static bool is_logging_lambda_form_invokers()              NOT_CDS_RETURN_(false);
 
+  // Can we skip some expensive operations related to modules?
+  static bool   is_using_optimized_module_handling()         { return CDS_ONLY(_is_using_optimized_module_handling) NOT_CDS(false); }
+  static void stop_using_optimized_module_handling()         NOT_CDS_RETURN;
 
   // CDS archived heap
-  static bool      is_dumping_heap()                         NOT_CDS_JAVA_HEAP_RETURN_(false);
-  static void disable_dumping_full_module_graph(const char* reason = nullptr) NOT_CDS_JAVA_HEAP_RETURN;
-  static bool      is_dumping_full_module_graph()            NOT_CDS_JAVA_HEAP_RETURN_(false);
-  static void disable_loading_full_module_graph(const char* reason = nullptr) NOT_CDS_JAVA_HEAP_RETURN;
-  static bool      is_loading_full_module_graph()            NOT_CDS_JAVA_HEAP_RETURN_(false);
-
+  static bool   is_dumping_heap()                            NOT_CDS_JAVA_HEAP_RETURN_(false);
+  static void stop_dumping_full_module_graph(const char* reason = nullptr) NOT_CDS_JAVA_HEAP_RETURN;
+  static bool   is_dumping_full_module_graph()               { return CDS_ONLY(_is_dumping_full_module_graph) NOT_CDS(false); }
+  static void   stop_using_full_module_graph(const char* reason = nullptr) NOT_CDS_JAVA_HEAP_RETURN;
+  static bool     is_using_full_module_graph()                NOT_CDS_JAVA_HEAP_RETURN_(false);
```

Comment:
Please align this with line #90.

Reply:
Do you mean aligning like this: 

```
  static bool is_dumping_heap()                              NOT_CDS_JAVA_HEAP_RETURN_(false);
  static void stop_dumping_full_module_graph(const char* reason = nullptr) NOT_CDS_JAVA_HEAP_RETURN;
  static bool is_dumping_full_module_graph()                 { return CDS_ONLY(_is_dumping_full_module_graph) NOT_CDS(false); }
  static void stop_using_full_module_graph(const char* reason = nullptr) NOT_CDS_JAVA_HEAP_RETURN;
  static bool is_using_full_module_graph()                   NOT_CDS_JAVA_HEAP_RETURN_(false);
```

I think this is more messy, and you can't easily tell that these four functions are all related to the same feature (full_module_graph).

Reply:
I wanted line 94 to align the "using_full_module_graph" part with the function at line 93.

Reply:
In that case, it looks good.